app/ledger.js

```diff
@@ -1291,11 +1313,8 @@ var syncWriter = (path, obj, options, cb) => {
 
 var pathName = (name) => {
   var parts = path.parse(name)
-  var basePath = process.env.NODE_ENV === 'test'
-    ? path.join(process.env.HOME, '.brave-test-ledger')
-    : app.getPath('userData')
 
-  return path.join(basePath, parts.name + parts.ext)
+  return path.join(app.getPath('userData'), parts.name + parts.ext)
```

Comment:
this is probably ok as of 3c1e03c93c279319b3a75d5d1f91cca56a09b733 now that i look at it, but there is some weirdness going on with tests right now